kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -44,11 +53,6 @@ public Kroxylicious(String deploymentNamespace) {
         this.containerImage = kroxyUrl + Environment.KROXY_VERSION;
     }
 
-    private void createDefaultConfigMap(String clusterName) {
```

Comment:
I can’t remember what was discussed but is it too early to remove testing of manual deployments? 

Sure at some point the operator will be our only supported solution but are we there yet?

Reply:
I understood that we want to remove that part and reviewing the Kroxylicious sync notes we said on the 4th of March that "System tests will target operator use-cases only."

Here we are targeting only the plain use cases, with no filters at all.

Reply:
I'm with @franvila on this one.  

I think we value two things 
* deployment to bare metal 
* deployments to Kubernetes with the Operator.

We've decided that the system test suite needs to focus exclusively on Kubernetes. .  I think we'd all prefer that had some system test coverage for bare metal, but we haven't the bandwidth for both.

I don't think we care about manual deployments to Kube any more.   The Operator will soon have feature parity with what we achieved with manual deployments anyway, so I think it'll be a straight swap.

I think trying to keep both working might actually slow us down, too.

Reply:
> Here we are targeting only the plain use cases, with no filters at all.

No, we should be able to reimplement RecordEncryptionST in terms on a Kafka Protocol Filter CR.  I think the Operator is sufficiently functional already for this to just work.

Reply:
> No, we should be able to reimplement RecordEncryptionST in terms on a Kafka Protocol Filter CR. 

But it would be best to migrate over a series of PRs, of course.

Reply:
sure

Reply:
Created #1996 to cover Record Encryption tests